web-console/console-config.js

```diff
@@ -19,4 +19,7 @@
 window.consoleConfig = {
   "exampleManifestsUrl": "https://druid.apache.org/data/example-manifests.tsv"
   /* future configs may go here */
+  "defaultQueryContext": {
+    "priority": -1
+  }
```

Comment:
just realized it's probably not be a good idea to set timeout to 30000 by default since we only did that for experimental purposes

Reply:
Let's put it back actually. we have already announced that we aren't allowing queries that run more than 30 secs from druid console.

Reply:
Oopz somehow thought the default is 30000 which is 30 sec and I put 300000 which is 5 min... It's actually the other way around 🤣